sdk/storage/storage-blob/package.json

```diff
@@ -24,7 +24,7 @@
     "build:autorest": "autorest ./swagger/README.md --typescript --use=@microsoft.azure/autorest.typescript@5.0.1",
     "build:es6": "tsc -p tsconfig.json",
     "build:nodebrowser": "rollup -c 2>&1",
-    "build:js-samples": "npm run clean && npm run build:es6 && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build:node-clean": "npm run clean && npm run build:es6 && cross-env ONLY_NODE=true rollup -c 2>&1",
```

Comment:
minor: Personally, I think even the original naming js-sample is fine for me, considering it's peer to ts-sample. @Xiaoning Liu, any comments on this?

Reply:
As mentioned in the linked issue, this script isn’t really building any JS samples (they are JS already. We want a better name to describe this script.